# HG changeset patch
# User Stefan Mahr <dac922@gmx.de>
# Date 1383310053 -3600
#      Fri Nov 01 13:47:33 2013 +0100
# Node ID 21e14380c3ad2cf98cfef8cafe52775fe0f11027
# Parent  0187ed94862752016bc628af88a8e851043c8e3e
maint: Add missing space

* libinterp/parse-tree/pt-pr-code.cc (tree_print_code::visit_do_until_command):
Add missing space.

diff --git a/libinterp/parse-tree/pt-pr-code.cc b/libinterp/parse-tree/pt-pr-code.cc
--- a/libinterp/parse-tree/pt-pr-code.cc
+++ b/libinterp/parse-tree/pt-pr-code.cc
@@ -1151,17 +1151,17 @@ tree_print_code::visit_do_until_command 
 
       decrement_indent_level ();
     }
 
   print_indented_comment (cmd.trailing_comment ());
 
   indent ();
 
-  os << "until";
+  os << "until ";
 
   tree_expression *expr = cmd.condition ();
 
   if (expr)
     expr->accept (*this);
 
   newline ();
 }
